src/script/components/Avatar/AvatarBadge.tsx

```diff
@@ -25,24 +25,36 @@ import {STATE} from '.';
 
 export interface AvatarBadgeProps {
   state: STATE;
+  iconSize?: string;
 }
 
-const AvatarBadge: React.FunctionComponent<AvatarBadgeProps> = ({state}) => {
+const AvatarBadge: React.FunctionComponent<AvatarBadgeProps> = ({state, iconSize = '16px'}) => {
   const icons: Record<string, string> = {
     [STATE.PENDING]: '\\e165',
     [STATE.BLOCKED]: '\\e104',
   };
+
+  const defaultBackgroundColor = 'rgba(0, 0, 0, .56)';
+  const backgroundColor: Record<string, string> = {
+    [STATE.BLOCKED]: 'var(--white)',
+  };
+
+  const defaultColor = '#fff';
```

Comment:
Try to use variables instead :)
```suggestion
  const defaultColor = 'var(--white)';
```